### ISSUE TITLE
zoom-us can't change pulseaudio sinks

### ISSUE THREAD

**Issue by lovesegfault** [NixOS/nixpkgs#80042]
**Describe the bug**
When I am in a Zoom conference call I cannot, using `pavucontrol` or `pacmd` change it's sink from my laptop's speakers to my headphones.

**To Reproduce**
1. Start Zoom and enter a meeting
2. Connect headphones
3. Attempt to set Zoom's sink using `pavucontrol`

**Expected behavior**
I'd expect to be able to move Zoom across sinks, like I can any other application (Firefox, for example, works fine.)

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.5.2, NixOS, 20.03pre212208.8130f3c1c2b (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.2`
 - channels(root): `"nixos-20.03pre212208.8130f3c1c2b"`
 - channels(bemeurer): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: zoom-us
# a list of nixos modules affected by the problem
module:
```

**Comment by worldofpeace** [comment]
@lovesegfault I've had this happen before office hours once (basically panic), it seems there's settings inside zoom to change it that work. The view  `Settings -> Audio` .

**Comment by lovesegfault** [comment]
Oh, If I set it to `System Default` then it seems to just obey PulseAudio!

**Comment by worldofpeace** [comment]
Hmm, I see `Same as system`(maybe that's different in another version).
I don't understand why that's not a default.

**Comment by lovesegfault** [comment]
Yeah, definitely bizarre behavior to override system defaults wrt. audio outputs

**Comment by mkg20001** [comment]
I think this is basically "works as intended" since you can specify the input/output in the app and then it tries to keep it that way per it's code

**Comment by mkg20001** [comment]
> The view Settings -> Audio .

The arrows next to the microphone/speaker in the meeting view should reveal the same settings

![image](https://user-images.githubusercontent.com/7735145/74542717-63dcb300-4f44-11ea-8aa0-9f49e4ffe2d9.png)

**Comment by lovesegfault** [comment]
Alright, closing this issue since I was just me not knowing how to use computers. Thanks for the help everyone!

**Comment by romildo** [comment]
I am also having this issue.

**Comment by mkg20001** [comment]
It seems to be application code releated AFAIK since it changes it back by itself... so "works as intended"?

**Comment by knarrff** [comment]
> I think this is basically "works as intended" since you can specify the input/output in the app and then it tries to keep it that way per it's code

I don't see it that way. The in-app settings should not be the only place where users can change the PA settings.

Also, if set to 'as system', this uses the system-default, which isn't necessarily the device you want for Zoom, and also that cannot be changed in pavucontrol, so at the moment, the only way to change the device is in Zoom itself, which is contrary to most other applications and limits the control people have over their system.

**Comment by miniBill** [comment]
Wrt using a monitor (which Zoom doesn't allow by default), a workaround is creating a virtual source with `module-remap-source`:
```
pactl load-module module-remap-source source_name=Remap-Source master=Virtual-Speaker.monitor
```

**Comment by katsar0v** [comment]
Any update on this issue?
